docs/using/frameworks/index.rst

```diff
@@ -23,8 +23,10 @@ streamline the process of adding oso to your application.
 What do you think?
 ==================
 We are working on updating our documentation with these integrations. In the
-meantime, **let us know your preferred framework/ORM** by using our chat on the bottom right
-to send us a Slack message.
+meantime, **let us know your preferred framework/ORM** in our 
+`Github repository <https://github.com/osohq/oso>`_,
```

Comment:
I don't this this is clear what to do right now, how would somebody tell us this?

Reply:
Good point. 

Opening an issue? Voting on issues? We can do [a poll](https://github.com/apex/gh-polls) later perhaps?